src/ovm/ObolValidatorManager.sol

```diff
@@ -203,14 +206,14 @@ contract ObolValidatorManager is OwnableRoles {
 
   /// Distributes target token inside the contract to recipients
   /// @dev pushes funds to recipients
-  function distributeFunds() external payable {
+  function distributeFunds() external {
     _distributeFunds(PUSH);
   }
 
   /// Distributes target token inside the contract to recipients
-  /// @dev backup recovery if any recipient tries to brick the OWRecipient for
+  /// @dev backup recovery if any recipient tries to brick the OVM for
   /// remaining recipients
-  function distributeFundsPull() external payable {
```

Comment:
What's the gas difference in removing `payable` if its like >500 gas i suggest we simply keep it

Reply:
okay only 21 gas in difference, not worth the confusion cause its no longer the hot path

Reply:
Discussed internally, `payable` has been removed.